build.sbt

```diff
@@ -19,6 +19,11 @@ lazy val buildSettings = Seq(
     homepage := Option(url("http://47deg.github.io/github4s/")),
     organizationHomepage := Option(new URL("http://47deg.com")),
     scalaVersion := "2.11.8",
+    crossScalaVersions := Seq("2.10.6", scalaVersion.value),
```

Comment:
Could you please solve this issue by the way?
https://github.com/47deg/github4s/issues/40

`crossScalaVersions := Seq("2.10.6", "2.11.8", "2.12.1")` would be great ;) . Thanks!